elements/jsonform/jsonform.stories.js

```diff
@@ -20,7 +20,7 @@ export default {
   render: (args) => html`
     <eox-jsonform
       .schema=${args.schema}
-      .startVals=${args.startVals}
+      .value=${args.startVals}
```

Comment:
Let's please also rename this in the storybook args, so `args.value` instead of `args.startVals`

Reply:
Resolved in https://github.com/EOX-A/EOxElements/pull/737/commits/f6d29b84aa3a6c6cc3d35b2c7cfbc76c0bd15970